src/ExtendedSystem.h

```diff
@@ -62,6 +72,15 @@ class ExtendedSystem  {
     double cellAngle[BOX_TOTAL][3];
     // Check to see if xsc is defined
     bool hasCellBasis[BOX_TOTAL];
+    // Stores the binary coordinates of both boxes
+    std::vector<XYZ> binaryCoor;
+    // For Loading 
+    int numAtoms;
+    int numAtomsInBox[BOX_TOTAL];
+    int boxStart[BOX_TOTAL];
+
+    // Stores the binary velocities of both boxes
+    std::vector<XYZ> binaryVeloc;
```

Comment:
Make sure to clear the the vectors after being used. We dont want to store data that we dont use through the simulation. We should save memory as much as we can.

Reply:
Cleared through clear -> allocate -> clear pattern
    binaryCoor.clear();
    binaryCoor.resize(pdb.atoms.beta.size());
    ReadCoordinate(pdb, inputFiles, molLookup, mols);
    UpdateCoordinate(pdb, inputFiles, molLookup, mols);    
    UpdateMinMaxAtoms(pdb, inputFiles, molLookup, mols);
    binaryCoor.clear();

Reply:
Same for vel